st2client/st2client/client.py

```diff
@@ -37,7 +37,7 @@
 
 class Client(object):
     def __init__(self, base_url=None, auth_url=None, api_url=None, api_version=None, cacert=None,
-                 debug=False, token=None):
+                 debug=False, token=None, apikey=None):
```

Comment:
I was under the impression it would be inconsistent with variable name in some other place, but I went and checked and could not find one. So here you go.